great_expectations/rule_based_profiler/types/data_assistant_result/volume_data_assistant_result.py

```diff
@@ -59,6 +59,8 @@ def plot(
 
         self.display(charts=charts, theme=theme)
 
+        return [chart.to_dict() for chart in charts]
```

Comment:
Should this be conditional? Should we always return the config?

I have concerns about polluting the notebook with config. Maybe a boolean flag in the signature?

Reply:
Additionally, we can use `chart.to_json()` as an alternative (returns a string). Thoughts here @NathanFarmer ?

Reply:
We could have both `to_dict` and `to_json` available with boolean flag defaulted to `False`.

Reply:
I'd rather not. Let's pick .to_dict() if we are going this way; it is no effort to convert a dictionary to json if folks want it that way. What is the use case for either (or both?)